docs/flax.traverse_util.rst

```diff
@@ -0,0 +1,44 @@
+flax.traverse_util package
+============================
+
+.. currentmodule:: flax.traverse_util
+
+.. automodule:: flax.traverse_util
+
+Traversal objects
+-------------------
+
+.. autoclass:: Traversal
+    :members:
+
+.. autoclass::TraverseId
```

Comment:
Is this missing a space?

Reply:
But actually, I don't think we should document these -- I consider them internal implementation details. People really only use `ModelParamTraversal`

Reply:
So I asked Jonathan before adding those and he thought it would be better to document those. I don't mind either way, I also see your point (this is why I asked in the first place).